spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/RewriteViewCommands.scala

```diff
@@ -123,4 +132,17 @@ case class RewriteViewCommands(spark: SparkSession) extends Rule[LogicalPlan] wi
 
     collectTempViews(child)
   }
+
+  private object IsIcebergView {
```

Comment:
The `Is` for this name doesn't make sense because it doesn't produce a boolean. It produces a loaded v2 view. I don't even think that the v2 view needs to be an Iceberg view.

I think this should probably be renamed to `ResolvedView` and the other should be named `ResolvedIdentifier` or something. I also don't like the duplication across these two helpers. It seems like this is doing what the other rule does, but should instead be delegating to other rules.

Reply:
Also, could this `unapply` method be added to `ResolvedView`?

Reply:
I like those new names better. I've updated the existing one to `ResolvedIdent` as otherwise it clashes with the existing `ResolvedIdentifier`. Unfortunately both `unapply` methods can't be combined